client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -30,7 +29,7 @@ ToolbarButton.propTypes = {
   iconName: PropTypes.string.isRequired
 };
 
-const DataFilesToolbar = ({ scheme, api }) => {
+const DataFilesToolbar = ({ scheme, api, path }) => {
```

Comment:
__Possibly Superfluous Parameter__
It might be unnecessary to pass `path` if the `getFilePermissions` function was used, because that function is given [`permissionParams`](https://github.com/TACC/Core-Portal/blob/1aaffc9/client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js#L166) which has `files` which __may__ have the path to those files.

Reply:
I see what you're saying here! I have now removed this parameter and all instances of it from the tests, now re-working my unit test to accomodate this change.